client/src/assets/player/shared/hotkeys/peertube-hotkeys-plugin.ts

```diff
@@ -79,7 +79,7 @@ class PeerTubeHotkeysPlugin extends Plugin {
       // Fullscreen
       {
         // f key or Ctrl + Enter
-        accept: e => this.isNaked(e, 'f') || (!e.altKey && e.ctrlKey && e.key === 'Enter'),
+        accept: e => this.isNaked(e, 'F') || (!e.altKey && e.ctrlKey && e.key === 'Enter'),
```

Comment:
I think `isNaked` should be in charge to do a `.toUpperCase()` of `f` so the caller doesn't have to think about casing

Reply:
Done in https://github.com/Chocobozzz/PeerTube/pull/5684/commits/a8a6504c3c85507b68e9aa91cf5b756e66084128